source/backend/apimodels/Models/Concepts/Project/ProjectPersonMap.cs

```diff
@@ -0,0 +1,27 @@
+using Mapster;
+using Pims.Api.Models.Base;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts.Project
+{
+    public class ProjectPersonMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsProjectPerson, ProjectPersonModel>()
+                .Map(dest => dest.Id, src => src.ProjectPersonId)
+                .Map(dest => dest.ProjectId, src => src.ProjectId)
+                .Map(dest => dest.Person, src => src.Person)
+                .Map(dest => dest.PersonId, src => src.PersonId)
+                .Map(dest => dest.Project, src => src.Project)
+                .Inherits<Entity.IBaseEntity, BaseConcurrentModel>();
+
+            config.NewConfig<ProjectPersonModel, Entity.PimsProjectPerson>()
+                .Map(dest => dest.ProjectPersonId, src => src.Id)
+                .Map(dest => dest.ProjectId, src => src.ProjectId)
+                .Map(dest => dest.PersonId, src => src.PersonId)
+                .Map(dest => dest.Person, src => src.Person)
+                .Inherits<BaseConcurrentModel, Entity.IBaseEntity>();
```

Comment:
question - I believe our pattern when we map models into EF entities is that we keep the ID but do not map child entities to avoid accidental updates to the children - right?

I see here that we correctly map the PersonId so that we can point to a new existing person but I also see the mapping of `.Map(dest => dest.Person, src => src.Person)` - shouldn't we take this out here?